<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>UISubclassWizard</class>
 <widget class="QDialog" name="UISubclassWizard" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SubClass Wizard</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item row="2" column="3" >
    <widget class="QPushButton" name="pbForward" >
     <property name="minimumSize" >
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Next</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/Icons/Icons/buttonnext.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QStackedWidget" name="swPages" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageMain" >
      <property name="toolTip" >
       <string>Class</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item row="2" column="0" >
        <widget class="QLabel" name="lGetWindowManager" >
         <property name="text" >
          <string>Window Manager :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="lGetParent" >
         <property name="text" >
          <string>Parent Class :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QComboBox" name="cbGetWindowManager" >
         <item>
          <property name="text" >
           <string>(all)</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>win32</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>unix</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>mac</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QComboBox" name="cbGetParent" >
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageHeader" >
      <property name="toolTip" >
       <string>Header</string>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QTextEdit" name="teHeader" >
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSource" >
      <property name="toolTip" >
       <string>Source</string>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>5</number>
       </property>
       <property name="spacing" >
        <number>3</number>
       </property>
       <item>
        <widget class="QTextEdit" name="teSource" >
         <property name="lineWrapMode" >
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="4" >
    <widget class="QLabel" name="lInformations" >
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="text" >
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="pbBackward" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="minimumSize" >
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Back</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/Icons/Icons/buttonprevious.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="pbCancel" >
     <property name="minimumSize" >
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor" >
      <cursor>13</cursor>
     </property>
     <property name="text" >
      <string>Cancel</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/Icons/Icons/buttoncancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>UISubclassWizard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>177</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel" >
     <x>225</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
